src/ansys/pyensight/core/launcher.py

```diff
@@ -165,6 +168,38 @@ def stop(self) -> None:
         """
         return
 
+    def _find_ports_used_by_other_pyensight_and_ensight(self):
```

Comment:
@mariostieriansys LOL I never thought about this approach.   It's got me thinking about other cases now.O

Reply:
LOL I'd like to know what you are thinking about